docs/extra.md

```diff
@@ -0,0 +1,93 @@
+To configure Pelorus to monitor your application, you need 3 information:
+
+* Which Hosting Service(s) for Version Control System (**HSVCS**) your application uses? (Ex.: GitHub)
+* Which Issue Tracker System(s) (**ITS**) your application uses? (Ex.: Jira)
+* Which OpenShift Namespace(s) (**NS**) your application lives in?
+
+To check the list of supported HSVCS and ITS by Pelorus, check [Supported data sources](#supported-data-sources).
+
+## Supported data sources
+
+| Provider      | Supported data sources |
+| ----------- | ----------- |
+| Git Providers      | <font size="6">:simple-github: :simple-bitbucket: :simple-gitea: :simple-gitlab: :simple-azuredevops:</font>   |
+| Issue Trackers      | <font size="6">:simple-jirasoftware: :simple-github: Service Now</font>   |
+| Deployment   | <font size="6">:simple-redhatopenshift:</font>        |
+
+### Git Providers
+
+<font size="5">
+
+- GitHub :simple-github:
+- Bitbucket :simple-bitbucket:
+- Gitea :simple-gitea:
+- GitLab :simple-gitlab:
+- Azure DevOps :simple-azuredevops:
+
+</font>
+
+### Issue Trackers
+
+<font size="5">
+
+- Jira :simple-jirasoftware:
+- GitHub :simple-github:
+- Service Now
```

Comment:
maybe we can get the logo from here https://www.servicenow.com/company/servicenow-logo.html

Reply:
created an issue, we can look this later